static/robots.txt

```diff
@@ -0,0 +1,17 @@
+Sitemap: https://datacommons.org/CongressionalDistrict.0.txt
```

Comment:
can we serve these from datacommons.org/sitemap? or do they need to be served from the root?

Reply:
Yeah, that was an error. updated to server from /sitemap